spec/fixtures/generate_external_certs.sh

```diff
@@ -0,0 +1,337 @@
+#!/bin/bash
+
+# This script re-generates all of the keys and certs within the external_certs directory.
+# The files there are used by the spec/features/settings/external_cert_feature_spec test
+
+function init() {
+  mkdir -p /tmp/ca_gen/csr
```

Comment:
It may make more sense to put all the mkdir commands below the variable setting, and mkdir based on the variables, rather than duplicating the paths.